modules/bankfeed/api/bankfeed.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import BankData from '../data/bandata';
+
```

Comment:
This file will need to be imported into app.ts and will need swagger.json updating with your endpoint data.

Reply:
@ageddesi 
 This part i am not able to understand, although i have implemented all again as you told me pleas have a look

Reply:
Thanks for the changes, if you look in app.ts you will see how the other route files are imported. If you can do this for this endpoint as well.

Do you also know how to update the swagger.json file with your new endpoints as well?

Reply:
No i do not have any idea about this.
I have tried what i could under from app.ts file regarding importing the modules do let me know if any other chnges

Reply:
@ageddesi 
Hii if my commit is fine enough then pleas do merge or let me know the chnges which i have to made 
And also Please give some more issues on which i can work for this hactober